bitnami/influxdb/values.yaml

```diff
@@ -51,45 +51,60 @@ clusterDomain: cluster.local
 ##
 architecture: standalone
 
-## Database to be created on first run
-## ref: https://github.com/bitnami/bitnami-docker-influxdb#creating-a-database-on-first-run
+## Authentication parameters
 ##
-database: my_database
-
-## Enable/disable authentication
-## ref: https://github.com/bitnami/bitnami-docker-influxdb#allowing-empty-passwords
-##
-authEnabled: true
-
-## InfluxDB(TM) admin credentials
-##
-adminUser:
-  name: admin
-  pwd:
-  usePasswordFile: false
-## InfluxDB(TM) credentials for user with 'admin' privileges on the db specified at 'database' parameter
-##
-user:
-  name:
-  pwd:
-  usePasswordFile: false
-## InfluxDB(TM) credentials for user with 'read' privileges on the db specified at 'database' parameter
-##
-readUser:
-  name:
-  pwd:
-  usePasswordFile: false
-## InfluxDB(TM) credentials for user with 'write' privileges on the db specified at 'database' parameter
-##
-writeUser:
-  name:
-  pwd:
-  usePasswordFile: false
-
-## Secret with InfluxDB(TM) credentials
-## NOTE: This will override the users/passwords defined at adminUser, user, readUser and writeUser
-##
-# existingSecret:
+auth:
+  ## Variable to keep compatibility with influxdb v1
+  enabled: true
+  ## InfluxDB(TM) admin credentials
+  ##
+  admin:
+    username: admin
+    password:
+    token:
+    usePasswordFile: false
```

Comment:
Can I suggest using a single `usePasswordFile`? I mean using `auth.usePasswordFiles` instead of `auth.admin.usePasswordFile`, `auth.user.usePasswordFile`, `auth.readUser.usePasswordFile`, and `auth.writeUser.usePasswordFile`
This is what we did in MariaDB (check the chart) for simplicity and since it's very unlikely that you want to have some secrets as env. vars while others as files.